rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-12-15
```

Comment:
Be consistent with Muta. https://github.com/nervosnetwork/muta/blob/master/rust-toolchain

Otherwise, I'm not sure if weird problems will occur.

Reply:
Done